src/components/MemberInviteList.js

```diff
@@ -0,0 +1,179 @@
+    const {translate} = useLocalize();
+    const [personalDetails, setPersonalDetails] = useState([]);
+    const [usersToInvite, setUsersToInvite] = useState([]);
+
+    const [searchTerm, setSearchTerm] = useState('');
+
+    const getSections = () => {
+        const sections = [];
+        let indexOffset = 0;
+
+        sections.push({
+            title: undefined,
+            data: selectedOptions,
+            shouldShow: true,
+            indexOffset,
+        });
+        indexOffset += selectedOptions.length;
+
+        // Filtering out selected users from the search results
+        const selectedLogins = _.map(selectedOptions, ({login}) => login);
+        const personalDetailsWithoutSelected = _.filter(personalDetails, ({login}) => !_.contains(selectedLogins, login));
+        const personalDetailsFormatted = _.map(personalDetailsWithoutSelected, (personalDetail) => OptionsListUtils.formatMemberForList(personalDetail, false));
+
+        sections.push({
+            title: translate('common.contacts'),
+            data: personalDetailsFormatted,
+            shouldShow: !_.isEmpty(personalDetailsFormatted),
+            indexOffset,
+        });
+        indexOffset += personalDetailsFormatted.length;
+
+        _.each(usersToInvite, (userToInvite) => {
+            const hasUnselectedUserToInvite = !_.contains(selectedLogins, userToInvite.login);
+
+            if (hasUnselectedUserToInvite) {
+                sections.push({
+                    title: undefined,
+                    data: [OptionsListUtils.formatMemberForList(userToInvite)],
+                    shouldShow: true,
+                    indexOffset: indexOffset++,
+                });
+            }
+        });
+
+        return sections;
+    };
+
+    const sections = didScreenTransitionEnd ? getSections() : [];
+
+    useEffect(() => {
+        setSearchTerm(SearchInputManager.searchInput);
+    }, []);
+
+    useEffect(() => {
+        const newUsersToInviteDict = {};
+
+        const inviteOptions = OptionsListUtils.getMemberInviteOptions(props.personalDetails, betas, searchTerm, excludedUsers);
+
+        // Update selectedOptions with the latest personalDetails information
+        const detailsMap = {};
+        _.forEach(inviteOptions.personalDetails, (detail) => (detailsMap[detail.login] = OptionsListUtils.formatMemberForList(detail, false)));
+        const newSelectedOptions = [];
+        _.forEach(selectedOptions, (option) => {
+            newSelectedOptions.push(_.has(detailsMap, option.login) ? {...detailsMap[option.login], isSelected: true} : option);
+        });
+
+        const userToInvite = inviteOptions.userToInvite;
+
+        // Only add the user to the invites list if it is valid
+        if (userToInvite) {
+            newUsersToInviteDict[userToInvite.accountID] = userToInvite;
+        }
+
+        setUsersToInvite(_.values(newUsersToInviteDict));
```

Comment:
@tienifr You can replace the above code with only one line, just like in `RoomInvitePage`.

```js
setUserToInvite(inviteOptions.userToInvite);
```

`inviteOptions.userToInvite` is always a single object so there's no point in creating the `newUsersToInviteDict` only to later call `_.values(newUsersToInviteDict`. This dictionary is a remnant of https://github.com/barros001/expensify-app/commit/cf357fb70e66b9f5255db39c45fbaa0beef9f0a4

Reply:
@tienifr I created a PR against your branch for the above:

https://github.com/tienifr/App/pull/33